straxen/plugins/daqreader.py

```diff
@@ -30,6 +30,7 @@ class DAQReader(strax.Plugin):
     dtype = strax.record_dtype()
     parallel = 'process'
     rechunk_on_save = False
+    compressor = 'lz4'
```

Comment:
See the discussion in https://github.com/AxFoundation/strax/pull/227: this is necessary for now but hopefully we can find another solution eventually.